src/reader.rs

```diff
@@ -22,15 +22,15 @@ use std::fmt::{Display, Formatter};
 use crate::types::Str;
 /// Configures and constructs new instances of [Reader].
 pub struct ReaderBuilder {
-    // This will be set to to `Some` catalog when the reader builder is created with catalog information.
-    // Default value for this field will be set to `None`.
```

Comment:
🗺️ This field used to be an `Option`. I've updated the comment in the constructor.